plugins/rollbar/src/hooks/useCatalogEntity.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useAsync } from 'react-use';
+import { useApi } from '@backstage/core';
+import {
+  catalogApiRef,
+  useEntityCompoundName,
+} from '@backstage/plugin-catalog';
+
+export function useCatalogEntity() {
+  const catalogApi = useApi(catalogApiRef);
+  const { namespace, name } = useEntityCompoundName();
+
+  const { value: entity, error, loading } = useAsync(
+    () => catalogApi.getEntityByName({ kind: 'Component', namespace, name }),
```

Comment:
@shmidt-i @Rugvip this code also relates to #1947 - we should settle on a way for the catalog wrapper to define a sensible root route format for the catalog (e.g. either `/catalog/default/component/my-component` or `/catalog/component%3Amy-component` or whatever) and then make sure there's a simple hook to get `{ kind, namespace, name }` and one to get `[entity, loading, error]`.

@andrewthauer no action on your part, this is fine in the current state of things